app/models/foreman_fog_proxmox/proxmox_vm_commands.rb

```diff
@@ -39,25 +39,32 @@ def create_vm(args = {})
       raise ::Foreman::Exception, format(N_('invalid vmid=%<vmid>s'), vmid: vmid) unless node.servers.id_valid?(vmid)
 
       image_id = args[:image_id]
-      name = args[:name]
       remove_volume_keys(args)
-      parsed_args = parse_typed_vm(args, type)
       if image_id
+        poolid = args[:pool] if args.key?(:pool)
         vm = clone_from_image(image_id, vmid)
-        options = {}
-        options[vm.container? ? :hostname : :name] = name
-        vm.update(parsed_args.merge(options))
+        vm.update(compute_clone_attributes(args, vm.container?, type))
+        update_pool(vm, poolid) if poolid
```

Comment:
```suggestion
        update_pool(vm, args[:pool]) if args.key?(:pool)
```
so we can drop line 44